scripts/TVs/add_new_media.py

```diff
@@ -66,6 +66,7 @@ def get_media_url():
 
 
 def add_new_media(username=None, tv=None):
+    students_to_refresh = []
```

Comment:
This code only ever handles a single student, so I don't think this list will ever contain more than a single student, because the video refresh is run after each single student.

Reply:
(No need to handle multiple students)

Reply:
this is a queue for refreshing slideshows at the end

Reply:
It will never add more than one user though.  There is no queue to queue.

Reply:
solved in [28db17b](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/130/commits/28db17b7a6ec34e3b00ff1e046e6cd1243271c5a)